docs/ref/fields.txt

```diff
@@ -0,0 +1,12 @@
+Fields Reference
+================
+
+``IsoDateTimeField``
+~~~~~~~~~~~~~~~~~~~~
+
+Extends ``django.forms.DateTimeField`` to parse ISO 8601 formated dates.
```

Comment:
Probably worth something like "additionally parse" - should make it clear that it's both ISO8601 *or* the existing formats.